src/KM_Controls.pas

```diff
@@ -350,6 +350,15 @@   TKMButtonFlat = class(TKMControl)
     procedure MouseUp(X,Y: Integer; Shift: TShiftState; Button: TMouseButton); override;
 
     procedure Paint; override;
+
+  end;
+
+
+  {FlatButton}
+  TKMButtonFlat = class(TKMButtonFlatCommon)

```

Comment:
Why is this needed? Every control has already OnClicked event that we could use.